### ISSUE TITLE
What changed between v4 and v5 in the ttf font?

### ISSUE THREAD

**Issue by Logikgate** [Templarian/MaterialDesign#5913]
Hello!

Thank you for this great project, we have been using it for a while now in our web app and our mobile iOS application. iOS only supports ttf fonts so we have been including the materialdesignicons-webfont.ttf file in our application and have, prior to version 5, had no trouble with iOS recognizing the font. 

When we upgraded to v5 iOS no longer recognizes the font. Do you know of anything that was changed between v4 and v5 with the ttf file besides the obvious that more glyphs were added?   We have also tried using the desktop ttf file since it looks like in v6 materialdesignicons-webfont.ttf will be removed but that did not work either.  Downgrading to v4 works just fine but we lose some of the icons we want to use.

Thank you for your help!

**Comment by Templarian** [comment]
Read more: 4901

TL;DR: All the codepoints got fixed so they start at `F0001` in the private use range. Probably need to update if you hardcoded the codepoints. Really should use vectors on mobile, the codepoints are terrible as the font is huge.

**Comment by Logikgate** [comment]
Ah thank you and I apologize for not finding that issue in my search. In the end it turned out to be an issue with the new code points being larger than 0xFFFF.
